### DISCUSSION TITLE
2023.11.24 第十六讲 感恩节， 你感恩吗？

### DISCUSSION THREAD

**Discussion by guttasingapore** [RoadSideUniversity/ClubHouse2024#40] [General]
感谢Shawn开房

**Comment by Lewisrxliu** [comment]
https://www.clubhouse.com/room/1dVSZ1OY2YTGkqkNn:gZdIX7d0-sq8f3nzAu70i1RBFbxMzizh9chGG-_edWI?utm_medium=ch_instant_invite&utm_campaign=lpYfcJN9MC3YSlNJ-t3K_A-996171

**Comment by guttasingapore** [comment]
https://en.wikipedia.org/wiki/Thanksgiving

**Comment by too** [comment]
[五分鐘日記：如何透過5分鐘改變你的人生（只需每天寫下五件事｜2021年日記） | SparkSine Blog](https://blog.sparksine.com/%E4%BA%94%E5%88%86%E9%90%98%E6%97%A5%E8%A8%98-%E6%AF%8F%E5%A4%A95%E5%88%86%E9%90%98%E6%94%B9%E8%AE%8A%E4%BD%A0%E7%9A%84%E4%BA%BA%E7%94%9F/)
https://blog.sparksine.com/五分鐘日記-每天5分鐘改變你的人生/

> 寫5分鐘日記是我在2020年最棒的一個新習慣之一。在寫下感恩事情的過程中，我學會了更細心地留意以及反思自己的生活、我明白到自己的一天發生了什麼事以及更珍惜所擁有的一切。

clubhouse上Mandy主持的感恩日记房，定期开，快速轻松聊
https://www.clubhouse.com/house/gratitude-daily-%E6%84%9F%E6%81%A9%E6%97%A5%E8%A8%98
